java/testing/testing-ii/testing-private-methods-and-fields.md

```diff
@@ -36,12 +33,16 @@ User john = new User("John");
 // which has a single argumet of type String
 // "new Class[] { String.class }""
 Method toTest = User
+  .getClass
   .getDeclaredMethod("checkPassHash",
     new Class[] { String.class });
 // make it accessible
 toTest.setAccessible(true);
```

Comment:
we should mention that this is only needed if you want to invoke a `private` method